Agreements/Internal/TCCOW_TCCPT.txt

```diff
@@ -0,0 +1,5 @@
+COPX:EGBB:*:COWLY:*:*:London TC COWLY:London TC CPT:19000:*:^COWLY
+COPX:EGBE:*:COWLY:*:*:London TC COWLY:London TC CPT:19000:*:^COWLY
```

Comment:
```suggestion
COPX:EGBB:*:COWLY:*:*:London TC COWLY:London TC CPT:19000:*:COWLY
COPX:EGBE:*:COWLY:*:*:London TC COWLY:London TC CPT:19000:*:COWLY
```

Reply:
Now changed in the upcoming ALD and vMATS releases - will make little difference practically so no plans for a procedure change post.

![image](https://user-images.githubusercontent.com/12017284/115233110-6d951400-a10f-11eb-8b0f-e159eb60a218.png)